src/pages/Main/Main.css

```diff
@@ -5,20 +5,22 @@ body {
 }
 
 .pageContainer {
-    width: 100%;
-    height: 100%;
+    width: 100vw;
+    height: 100vh;
+    height: calc(var(--vh, 1vh) * 100);;
```

Comment:
isn't the first height specification ignored here?

Reply:
The first height setting is a backup in case the browser the person is using doesn't support calc.

Reply:
Got it, thanks

Reply:
There are 2 semis on line 10, that looks like a typo?

Reply:
yes thanks!